src/components/UASTViewer.js

```diff
@@ -1,18 +1,35 @@
 import React, { Component } from 'react';
 import styled from 'styled-components';
+import Node from './uast/Node';
+import {eventHandler} from './uast/EventHandler';
+import {font, background} from '../styling/variables';
 
 const Container = styled.div`
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  overflow: auto;
+  font-family: ${font.family.code};
+  font-size: ${font.size.large};
+  background: ${background.main};
+  line-height: ${font.lineHeight.large};
+  padding: 4px;
 `
 
 export default class UASTViewer extends Component {
+  constructor(props) {
+    super(props);
+    eventHandler.onNodeSelected = this.props.onNodeSelected;
```

Comment:
i'd move all the logic in eventhandler into the UASTViewer, it really makes no sense to have it outside when it's clearly the responsibility of this component.

If I'm understanding this well, EventHandler is just a singleton class that is used as a hack to not having to pass `onNodeSelected` down to all children, amirite? I'd rather have this done the right way.